app/networkapi/settings.py

```diff
@@ -45,7 +46,9 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env('DEBUG')
 
-ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+ALLOWED_HOSTS = os.getenv("ALLOWED_HOSTS").split(",")
```

Comment:
I would like to stick to one `env` pattern instead of two and `env` is a really good library for that. Maybe let's try to debug over vidyo?